### PR TITLE
Fixed internal error found by AY integration tests

### PR DESCRIPTION
**Pull request by mchf**
bnc#974910

additional info:
boot4

### CODE REVIEW

**Comment by jreidinger** [comment]
LGTM

File: src/modules/LanItems.rb [hunk 1]
```diff
@@ -2336,6 +2336,8 @@ def createS390Device(rule)
     #  @param [Array] list of hashes describing interfaces in AY profile
     #  @return [Array] list of hashes for udev rules
     def createUdevFromIfaceName(interfaces)
+      return [] if !interfaces || interfaces.empty?
```
Comment (jreidinger) [inline src/modules/LanItems.rb]: btw it will work if you also use `interfaces ||= []` line, but it is just matter of taste

**Comment by teclator** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for reviews

File: src/modules/LanItems.rb (hunk 1 above)
Comment (kobliha) [inline src/modules/LanItems.rb]: It's the fastest exit from the method, which easy to understand and independent on the rest of the code even in the future

Comment (jreidinger) [inline src/modules/LanItems.rb]: yes, but on other hand, sanitizing input allows you to e.g. log what happen (empty interfaces) and it is easier to see that empty interfaces is same as if there is no specific device type with id or bus identifier type.

**Pull request merged**